wcfsetup/install/files/lib/action/AJAXUploadAction.class.php

```diff
@@ -34,10 +35,10 @@ protected function sendResponse()
             parent::sendResponse();
         }
 
+        $response = new JsonResponse($this->response);
         // IE9 is mad if iframe response is application/json
-        \header('Content-type: text/plain');
-        echo JSON::encode($this->response);
+        $response = $response->withHeader('content-type', 'text/plain');
```

Comment:
Please just remove this non-sense, it is an old legacy hack for IE9 :D

Reply:
As explained in [the commit message](https://github.com/WoltLab/WCF/pull/4447/commits/4dfe137efc1fb1dfea1ae435335749218d13c2cb):

> The IE 9 fallback technically should no longer be required, but the (legacy)
> JavaScript still references the `isFallback` parameter.

I'd rather clean this up once the JavaScript is cleaned up as well to not mix 2 unrelated things.

Reply:
OK, I was under the false impression that the fallback was removed a while ago, because it is effectively unsupported.